packages/charts/src/components/common/components/Tooltip/TooltipContent.tsx

```diff
@@ -13,18 +13,24 @@ type TooltipContentV2Props = {
     entries: TooltipEntry[];
     title?: string;
     description?: string;
+    imageUrl?: string;
 };
 
-export const TooltipContent = ({ title, description, entries }: TooltipContentV2Props) => {
+export const TooltipContent = ({ title, description, imageUrl, entries }: TooltipContentV2Props) => {
```

Comment:
Why not go with a composition approach?
```tsx
<TooltipContent.Root>
  <TooltipContent.Title>My title</TooltipContent.Title>
  <TooltipContent.Body>
    <TooltipContent.Image src="" alt="" />
    <Text>Some content under the image</Text>
  </TooltipContent.Body>
</TooltipContent.Root>
```

Reply:
That's a good pattern(although I don't like the dot notation), but it's out of the scope of this PR